vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -48,42 +45,51 @@ If that is your desired layout, you can create your charts/components without pr
 
 
 ## Configure the grid
-To customize the grid arrangement, you can configure the `grid` parameter of the [`Layout`][vizro.models.Layout] model.
-The example below shows how the grid works and how to specify a valid one:
+To customize the grid arrangement, configure the `grid` parameter of the [`Layout`][vizro.models.Layout] model.
 
-```python title="Basic Example"
+The example below shows an example of a valid `grid`:
+
+```python title="Basic example"
 grid = [[0, 1],
         [0, 2]]
 ```
 
-- The `grid` needs to be provided as `List[List[int]]` (for example, `grid = [[0, 1], [0, 2]]`)
-- The integers in the `grid` correspond to the index of the chart/component inside the list of `components` provided to [`Page`][vizro.models.Page]
-- The number of integers in the `grid` needs to match the number of chart/components provided
-- Each sub-list corresponds to a grid row (for example, row 1 = `[0, 1]` and row 2 = `[0, 2]`)
+- The `grid` must be provided as `List[List[int]]` (for example, `grid = [[0, 1], [0, 2]]`).
+- The integers in the `grid` must be consecutive integers starting with 0 (for example, `0`, `1`, `2`).
+    - The integers correspond to the index of the chart/component inside the list of `components` provided to [`Page`][vizro.models.Page].
+    - The number of integers in the `grid` needs to match the number of chart/components provided.
+- Each sub-list corresponds to a grid row (in the example above, row 1 = `[0, 1]` and row 2 = `[0, 2]`)
 - Each element inside the sub-list corresponds to a grid column (for example, column 1 = `[0, 0]` and column 2 = `[1, 2]`)
-- The integers in the `grid` need to be consecutive integers starting with 0 (for example, `0`, `1`, `2`)
-- Each chart/component will take the entire space of its grid area (empty spaces are currently not enabled)
-- The area spanned by a chart/component in the grid must be rectangular
-- The grid can be arbitrarily large, allowing arbitrarily granular control of the grid:
-
-```python title="Advanced Example"
-grid=[[0, 1, 3, 4],
-      [2, 2, 3, 4]]
+
+- Each chart/component will take the entire space of its grid area (spaces are not enabled).
+- The area spanned by a chart/component in the grid must be rectangular.
+- The grid can be arbitrarily large, allowing arbitrarily granular control of the grid.
+
+
+## Vertical and horizontal stacking
+As described above, when no `Layout` is specified, components are presented **vertically** as a single-column stack. If you have three components, the default `Layout.grid` will be as follows:
+
+```python title="Default grid"
+grid = [[0], [1], [2]]
+```
```

Comment:
I know this formatting is a bit ugly, but I think it helps to understand the grid layout
```suggestion
```python title="Default grid"
grid = [
             [0], 
             [1],
             [2]
       ]
```
```